.github/workflows/lint.yml

```diff
@@ -7,6 +7,26 @@ on:
       - release/**
 
 jobs:
+  revive-lint:
+    name: Run lint with revive
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3
+      - uses: technote-space/get-diff-action@v6.1.0
+        with:
+          PATTERNS: |
+            **/*.go
+      - uses: actions/setup-go@v3
+        if: env.GIT_DIFF
+        with:
+          go-version: 1.19
+          cache: true
+      - name: revive lint action
+        uses: morphy2k/revive-action@v2
+        with:
+          path: "./..."
```

Comment:
how about adding it under golangci job? This will save us some CI cycles.

Reply:
Okay i will move revive lint into golangci job